sgx/untrusted/src/util/ecdh_dummy_server.c

```diff
@@ -16,7 +16,7 @@ int ecdh_dummy_server(unsigned char *client_pub,
                       unsigned char *client_pub_sig,
```

Comment:
Is ecdh_dummy_server.c (and ecdh_dummy_server.h) unneeded at this point? If so, I think we should probably go ahead and delete them.

Reply:
Yes, I can remove them.